### PR TITLE
fix: add nested message resource to database

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**
In monitoring API, we lost the resource which is the nested field of a message. Like below, we did not generate resource as `AlertPolicyCondition`

```
message AlertPolicy {
  option (google.api.resource) = {
    type: "monitoring.googleapis.com/AlertPolicy"
    pattern: "projects/{project}/alertPolicies/{alert_policy}"
    pattern: "organizations/{organization}/alertPolicies/{alert_policy}"
    pattern: "folders/{folder}/alertPolicies/{alert_policy}"
    pattern: "*"
  };
 // ... other messages
  message Condition {
    option (google.api.resource) = {
      type: "monitoring.googleapis.com/AlertPolicyCondition"
      pattern: "projects/{project}/alertPolicies/{alert_policy}/conditions/{condition}"
      pattern: "organizations/{organization}/alertPolicies/{alert_policy}/conditions/{condition}"
      pattern: "folders/{folder}/alertPolicies/{alert_policy}/conditions/{condition}"
      pattern: "*"
    };
   // ... other fields
  }
}
```
Now we put all messages in a stack, pop one at a time, if it has nested messages, push it into the stack until the stack is empty.

### CODE REVIEW

**Comment by xiaozhenliu-gg5** [comment]
1. add all proto files in`monitoring/v3` and update unit test,  update baseline data.
2. add nested fields to messages stack for all resources coverage.
3. `monitoring.googleapis.com/TimeSeries` seems missing in the proto, this needs file-level resource definition.

File: .circleci/config.yml [hunk 1]
```diff
@@ -139,8 +139,6 @@ jobs:
             npm run fix
             npm run compile
             npm run system-test
-            npm run docs
```
Comment (alexander-fenster) [inline .circleci/config.yml]: What's up with this - do they fail?

File: typescript/src/schema/api.ts [hunk 2]
```diff
@@ -163,24 +163,28 @@ function getResourceDatabase(
...
         m?.options?.['.google.api.resource'] as ResourceDescriptor | undefined,
-        `file ${fd.name} message ${property}`
+        `file ${fd.name} message ${messageName}`
       );
       allResourceDatabase.registerResource(
         m?.options?.['.google.api.resource'] as ResourceDescriptor | undefined,
-        `file ${fd.name} message ${property}`
+        `file ${fd.name} message ${messageName}`
       );
+      if (m.nestedType) {
```
Comment (alexander-fenster) [inline typescript/src/schema/api.ts]: in one line: `(m.nestedType ?? []).map(m => messagesStack.push(m));` (not sure if it's more readable though :) )

**Comment by alexander-fenster** [comment]
Thanks for catching this @xiaozhenliu-gg5!

File: typescript/src/schema/api.ts (hunk 2 above)
Comment (xiaozhenliu-gg5) [reply typescript/src/schema/api.ts]: yes sure, that makes sense to me! thanks!

File: .circleci/config.yml (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply .circleci/config.yml]: yes, there are 4 broken links.

**Pull request merged**